fs/io_zenfs.cc

```diff
@@ -215,25 +220,28 @@ Status ZoneFile::MergeUpdate(std::shared_ptr<ZoneFile> update, bool replace) {
   is_sparse_ = update->IsSparse();
   MetadataSynced();
 
+  linkfiles_.clear();
+  for (uint32_t i = 0; i < update->GetNrLinks(); i++) {
+    linkfiles_.push_back(update->GetLinkFilename(i));
+  }
```

Comment:
After this change, [this](https://github.com/westerndigitalcorporation/zenfs/pull/156/files#diff-620cbbece5179b69096a6f453042d59dffedcb5672fb0f4b7f343d68e664314aR203) line has no effect and should be removed.

Reply:
I did not get what you are referring to, the hyperlink points to a diff rather than a line.

Reply:
Sorry, I thought it retained the line info. I was trying to refer to `Rename()` earlier in the method.